### DISCUSSION TITLE
How to buy and have a bigger codespaces

### DISCUSSION THREAD

**Discussion by GinKun43** [community/community#127772] [Codespaces] [answered]
### Select Topic Area

Question

### Body

I wanna buy and have a 16 core with 32gb ram like this guy
![image](https://github.com/community/community/assets/76711500/2a4d505d-6b1c-46d4-9280-45dbd79f1973)
how can I have it and buy it?
Thanks!

**Comment by Akash1134** [comment, marked as answer]
Hi @GinKun43 ,

To upgrade your GitHub Codespaces, you need to follow these steps:

- Go to your GitHub settings.Click on "Codespaces" in the left sidebar.
- Adjust your machine type and select a larger size based on your needs.

Please note that charges apply for using Codespaces, and the cost depends on the machine type you select. You can find more information about pricing on the [GitHub pricing page](https://github.com/pricing) and our[ FAQ's](https://github.com/pricing/calculator).

Note that the resources you choose should depend on the requirements of your project. For most development work, an 8 core machine with 16 GB RAM should be more than sufficient. See [how to upgrade your Codespaces](https://docs.github.com/en/codespaces/developing-in-a-codespace/using-github-codespaces-in-visual-studio-code#changing-the-machine-type-in-visual-studio-code). 

Thanks!

**Reply by GinKun43** [reply]
![image](https://github.com/community/community/assets/76711500/4584fddf-b742-45cd-b5ce-5de0b96b3ec7)
I have tried the ways to upgrade that you showed and hope it will be successful
It took me almost a week to find some ways to upgrade my machine 
and I wrote a ticket to staff but it seems they haven't responded yet
Thank you Sir!

**Reply by GinKun43** [reply]
![image](https://github.com/community/community/assets/76711500/7b131808-bb89-4e55-98d6-e551c609fe66)
It was successful
just a few minutes ago 
I really appreciate your help
Thank you!
